src/Utils/generics.ts

```diff
@@ -261,10 +261,11 @@ export const fetchLatestWaWebVersion = async(options: AxiosRequestConfig<any>) =
 				responseType: 'json'
 			}
 		)
-		const version = result.data.currentVersion.split('.')
+		const versionData = result.data.currentVersion.split('.')
+		const version = [+versionData[0], +versionData[1], +versionData[2]] as WAVersion
 		return {
-			version: [+version[0], +version[1], +version[2]] as WAVersion,
-			isLatest: true
+			version: version,
+			isLatest:  isLatest(lastVersion, version)
 		}
```

Comment:
url always gives latest version of wa web

Reply:
Sure, but 

> I don't think this is suitable for merge

This update will help in making a decision whether to disconnect from WhatsApp and re-use the latest version (simulating the process of updating WhatsApp Web).

Reply:
In short, the function now makes it easy for the user to know whether the running instance is still the latest or not.
Anyway, I appreciate your point of view, and let's waiting for the other experts.